neuralet-distancing.py

```diff
@@ -3,16 +3,18 @@
 from multiprocessing import Process
 import threading
 from libs.config_engine import ConfigEngine
-import libs.pubsub
 import logging
 
 logger = logging.getLogger(__name__)
 
 
 def start_engine(config, video_path):
-    from libs.core import Distancing as CvEngine
-    engine = CvEngine(config)
-    engine.process_video(video_path)
+    if video_path:
+        from libs.core import Distancing as CvEngine
+        engine = CvEngine(config)
+        engine.process_video(video_path)
+    else:
+        logger.warning('Skipping CVEngine as video_path is not set in config file')
```

Comment:
Why do we have to handle this? I assume the start is called, and if it is not configured correctly the program has to throw some errors and exit. 
Also the message seems not to be logically true, if video_path is null, it does not mean that "video_path is not set in config file", it means whoever called this, has called it with a video_path which is not checked or whatever. 
Anyhow, due to changes in the architecture in near future, and due to the nature of the process-based design, just ignore what I wrote and do not change the code, I am writing these to remember them later in the new architecture :D

Reply:
When I wanted to work on the frontend, I needed the backend running, but processing the video used a lot of my CPU.